schema.sql

```diff
@@ -10,3 +10,30 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD species VARCHAR(255);
+
+/*Create owners table*/
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name VARCHAR(255),
+    age SMALLINT
+);
+
+/*Create species table*/
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    specie_name VARCHAR(255)
```

Comment:
- [ ] Kindly ensure that this column is named correctly as `name` like in the project requirements